examples/linearCylinderPlaneStrain/linearCylinderPlaneStrain.m

```diff
@@ -0,0 +1,149 @@
+%md```math 
+%md u_r(r) = Ar + \dfrac{B}{r} \quad \text{and} \quad u_z(z)  \\
+%md A = \dfrac{(1+\nu)(1-2\nu)R_i^2p_i}{E(R_e^2-R_i^2)}, \quad
+%md B = \dfrac{(1+\nu)R_i^2R_e^2p_i}{E(R_e^2-R_i^2)}
+%md```
+%md### Numerical solution
+%md Before defining the structs, the workspace is cleaned, the ONSAS directory is added to the path and scalar geometry and material parameters are defined:
+clear all, close all
+% add path
+addpath( genpath( [ pwd '/../../src'] ) ) ;
+% scalar parameters
+E = 1e6 ; nu = 0.3 ; p = 1 ; L = .75 ;
+%md
+%md
+%md### MEBI parameters
+%md
+%md#### materials
+%md The constitutive behavior of the material considered is isotropic linear elastic.
+%md Since only one material is considered, the structs defined for the materials contain only one entry:
+materials.hyperElasModel  = 'linearElastic' ;
+materials.hyperElasParams =  [ E nu ]       ;
+%md
+%md#### elements
+%md 
+%md In this model three kinds of elements are used: `triangle` for the solid, `edges` to add external loads and `nodes`. Since three kinds of elements are used, the struct has length 3: 
+elements(1).elemType = 'node';
+elements(2).elemType = 'edge';
+elements(2).elemCrossSecParams = L ;
+elements(3).elemType = 'triangle';
+elements(3).elemTypeParams = 2 ;
+elements(3).elemCrossSecParams = L ;
+%md where `elemCrossSecParams` field sets the thickness of the edge and `elemTypeParams` sets the plane strain triangle element.  
+%md
+%md#### boundaryConds
+%md Three BCs are considered, one corresponding to a load and two for displacements.
+%md The first two BCs constrain displacements in $x$ and $y$ local directions respectively:
+boundaryConds(1).imposDispDofs = [1] ;
+boundaryConds(1).imposDispVals = [0] ;
+boundaryConds(2).imposDispDofs = [3] ;
+boundaryConds(2).imposDispVals = [0] ;
+%md then the third BC introduced in local coordinates sets the radial internal pressure (here is used as a convention that compression to the surface is positive):
+boundaryConds(3).loadsCoordSys = 'local' ;
+boundaryConds(3).loadsTimeFact = @(t) t  ;
+boundaryConds(3).loadsBaseVals = [ p 0  0 0  0 0 ]  ;
+%md
+%md#### initialConds
+%md Any non-homogeneous initial conditions are considered, thereafter an empty struct is set:
+initialConds = struct();
+%md
+%md### Mesh
+%md The mesh is automatically generetd with GMSH software:
+%md
+%md```@raw html
+%md<img src="../../assets/linearCylinderPlaneStrain/meshCylinderPlaneStrain.png" alt="mesh plot" width="500"/>
+%md```
+%md
+%md the mesh is read using `meshFileReader.m` function
+[ mesh.nodesCoords, mesh.conecCell ] = meshFileReader( 'ring.msh' ) ;
+%md
+%md### Analysis parameters
+%md
+%md The Newton-Raphson method is employed with tight tolerances and 2 load steps. The ratio between `finalTime` and `deltaT` sets the number of load steps used to evaluate `boundaryConds(3).loadsTimeFact` function:  
```

Comment:
I suggest removing these tolerances and leaving the default ones... I would also not comment on the tolerances since the problem is linear and no iteration should be done...

Reply:
Done